include/interfacekernels/ADPotentialSurfaceCharge.h

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ADInterfaceKernel.h"
+
+/**
+ * DG kernel for interfacing diffusion between two variables on adjacent blocks,
+ * using the automatic differentiation system to calculate the Jacobian.
```

Comment:
Could you make this class description comment a bit more specific to this class? This looks like the same one from the `ADCoupledInterfacialSource` classes in MOOSE.